projects/storefrontlib/src/shared/utils/forms/form-utils.ts

```diff
@@ -22,4 +22,29 @@ export class FormUtils {
     const control: AbstractControl = form.get(formControlName);
     return control.invalid && (submitted || (control.touched && control.dirty));
   }
+
+  /**
+   * Marks all form fields as 'touched'
+   * Helpful when checking empty or invalid forms
+   * @param form Form with fields to mark
+   */
+  static markFormAsTouched(form: FormGroup): void {
+    Object.keys(form.controls).forEach(controlName => {
```

Comment:
Why not doing just `form.markAsTouched()`?

Reply:
fixed